ui/src/message_popup/message_popup_page.jsx

```diff
@@ -174,6 +157,31 @@ export default React.createClass({
         </div>
       </div>
     );
+  },
+
+  renderScaffoldingOptions() {
+    return (
+      <div>
+        <div style={{fontSize: 16, padding: 20}}>
+          <Toggle
+            label="With student cards"
+            labelPosition="right"
+            toggled={this.state.shouldShowStudentCards}
+            onToggle={this.onStudentCardsToggled} />
+          <Toggle
+            label="With feeback and revision"
+            labelPosition="right"
+            toggled={false}
+            disabled={true}  />
+          <Toggle
+            label="With hints available"
+            labelPosition="right"
+            toggled={false}
```

Comment:
This is where I would have added in the changes I made in my PR. That can be either done here or during the merge.